packages/lesswrong/lib/collections/posts/helpers.ts

```diff
@@ -188,7 +188,11 @@ export const postGetCommentCount = (post: PostWithCommentCounts): number => {
 export const postGetCommentCountStr = (post?: PostWithCommentCounts|null, commentCount?: number|undefined): string => {
   const count = commentCount !== undefined ? commentCount : post ? postGetCommentCount(post) : 0;
   if (!count) {
-    return "No comments";
+    if (isLWorAF) {
+      return "0 comments";
```

Comment:
This affects e.g. preview tooltips as well, just FYI

Reply:
Hrm. That seems like maybe we need to actually think more about this tomorrow.